ext/webidl/00_webidl.js

```diff
@@ -328,7 +328,7 @@
   };
 
   converters["unrestricted float"] = (V, opts) => {
```

Comment:
This `opts` seems unused now

Reply:
Ah, missed this one. `unrestricted double` already got `opts` turned to `_opts` but this one remains.